spec/models/student_account_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe StudentAccount, type: :model do
+
+  context 'associations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.